task-list/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> <%=@person.name%>'s' Incomplete Tasks: </h1>
+<ul id=person_task_list>
+  <% @remaining_tasks.each do |t| %>
+    <li>
+      Task: <%=t.name%>
```

Comment:
For readability, it is best practice to add a space after the = and before the last % so:
`<%= t.name %>`